From 0cee352e4ea2520d054a3be057143d4f699d9b9c Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:16:53 -0400
Subject: [alsa] usb-audio: Add missing kconfig dependecy

Message-id: <140972823573-79-patchset-jkysela@redhat.com>
Patchwork-id: 90779
O-Subject: [RHEL 7.1 PATCH 078/442] ALSA: usb-audio: Add missing kconfig dependecy
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The commit 44dcbbb1cd61 introduced the usage of bitreverse helpers but
forgot to add the dependency.  This patch adds the selection for
CONFIG_BITREVERSE.

Fixes: 44dcbbb1cd61 ('ALSA: snd-usb: add support for bit-reversed byte formats')
Reported-by: Fengguang Wu <fengguang.wu@intel.com>
Cc: <stable@vger.kernel.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 4fa71c1550a857ff1dbfe9e99acc1f4cfec5f0d0)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/usb/Kconfig b/sound/usb/Kconfig
index de9408b..e05a86b 100644
--- a/sound/usb/Kconfig
+++ b/sound/usb/Kconfig
@@ -14,6 +14,7 @@ config SND_USB_AUDIO
  select SND_HWDEP
  select SND_RAWMIDI
  select SND_PCM
+ select BITREVERSE
  help
    Say Y here to include support for USB audio and USB MIDI
    devices.
-- 
1.7.1